src/request/GetContentType.ts

```diff
@@ -1,14 +1,13 @@
 import { ContentType } from '@dreamit/graphql-server-base'
 
 export function getContentType(contentType?: string): ContentType {
-    if (contentType) {
-        if (contentType.includes(ContentType.graphql)) {
-            return ContentType.graphql
-        } else if (contentType.includes(ContentType.json)) {
-            return ContentType.json
-        } else if (contentType.includes(ContentType.urlencoded)) {
-            return ContentType.urlencoded
-        }
+    if (
+        contentType &&
+        (contentType.includes('application/graphql') ||
+            contentType.includes('application/json') ||
+            contentType.includes('application/x-www-form-urlencoded'))
+    ) {
+        return contentType as ContentType
```

Comment:
Shouldn't this function be part of `graphql-server-base`? Seems like `ContentType` is only really usable with this.

Reply:
I try to avoid putting functions with logic in graphql-server-base so it stays small and only provides the necessary code for extensions. ContentType as type (alone) allows the type check to ensure that only the provided values are used, this does not need the GetContentType function, e.g. in https://github.com/dreamit-de/graphql-testing/pull/64/files#diff-89be8437f27a5eb8d8cfc580e3e14d918d6d7679e6688a2aa620259c6820feebR27 .